aicsimageio/tests/readers/test_default_reader.py

```diff
@@ -3,14 +3,15 @@
 
 import numpy as np
 import pytest
+
 from aicsimageio import exceptions
 from aicsimageio.readers import DefaultReader
 
-from ..conftest import LOCAL, REMOTE, get_resource_full_path
+from ..conftest import get_resource_full_path, host
 from ..image_container_test_utils import run_image_read_checks
 
 
-@pytest.mark.parametrize("host", [LOCAL, REMOTE])
+@host
```

Comment:
Is this a new hook?

Reply:
Nah this is just generalized and made available in `conftest` because it is used all over the place. So nicer to keep it all in one place than redefine multiple times.